### DISCUSSION TITLE
Download User Whole Profile - Setting Question?

### DISCUSSION THREAD

**Discussion by Kuroo2021** [MalloyDelacroix/DownloaderForReddit#286] [Q&A] [answered]
Hello within the download area of the settings "Download Defaults" I Default [USER] and choose post limit to 1000. (Max) But I'm still not getting content that is only 3 months old and not older content for the use case I'm looking at. The user has other images posted but can't fully download them all. What settings would I need to turn on to just download all of the content from the user I added to the list?

**Comment by MalloyDelacroix** [comment, marked as answer]
If the settings have not been changed from their default (except for the max post setting), you should be set for what you are trying to do.  

If you have already run a download session on the list, make sure you set the custom date limit to a time before the oldest post you are wanting to download.  By default, this is set to the oldest possible post.  But after you run a download it will be updated to the newest post downloaded for each user/subreddit.

Make sure that the download videos, images, and gifs checkboxes are checked if you want to download each type of content.  

Ensure that the nsfw filter is set to your desired preference.  "Include" will download everything regardless of if it is nsfw or not.

If you intend to download the user multiple times to keep getting all of the posts, be sure that you are sorting the posts by NEW.  This will keep the date limit setting correct as you download multiple times.

It's possible in some circumstances that the user has made more than 1,000 posts, even in a short amount of time.  It could also be possible that the user has a lot of duplicate posts which will only be downloaded one time by default.  If you want to download duplicate posts, you can uncheck the "avoid duplicates" checkbox in the settings.

It's also possible that the user has deleted the content that some of the posts link to.  The thumbnails may still show up on reddit, but the link is dead.

If you have these settings set and still encounter the same problem, you can open an issue and attach your log file and we may be able to resolve it.

**Reply by Kuroo2021** [reply]
I was looking around and was not sure where the log file location is?  I check the application folder but didn't see anything. I check the download area from were and found a CSV file label as "Default", it also only has 1kb with of data in there. Any idea where the log file is?

**Reply by MalloyDelacroix** [reply]
C:/Users/[username]/AppData/Roaming/SomeGuySoftware/DownloaderForReddit

**Reply by Kuroo2021** [reply]
ty
